src/main/java/de/medizininformatik_initiative/process/data_sharing/spring/config/DicFhirClientConfig.java

```diff
@@ -103,10 +103,15 @@ public class DicFhirClientConfig
 	private String fhirStoreProxyPassword;
 
 	@ProcessDocumentation(processNames = {
-			"medizininformatik-initiativede_executeDataSharing" }, description = "The url of the oidc provider to request access tokens (token endpoint)", example = "http://foo.baz/realms/fhir-realm/protocol/openid-connect/token")
+			"medizininformatik-initiativede_executeDataSharing" }, description = "The base url of the oidc provider", example = "http://foo.baz/realms/fhir-realm/protocol/openid-connect/token")
```

Comment:
The given example URL is incorrect. It should be `http://foo.baz/realms/fhir-realm`.

Reply:
well spotted, thanks --> f27f003